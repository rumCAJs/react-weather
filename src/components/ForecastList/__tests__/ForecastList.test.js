import React from 'react'
import { ForecastList } from '../index'
import { getForecastData } from './data'
import renderer from 'react-test-renderer'

const data = getForecastData()
it('renders correctly', () => {
  const tree = renderer.create(<ForecastList forecast={data} />).toJSON()
  expect(tree).toMatchSnapshot()
})
