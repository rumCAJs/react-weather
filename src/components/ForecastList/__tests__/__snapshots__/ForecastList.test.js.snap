// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="styled__Heading-ev9rqb-1 nYOfy"
  >
    Forecast for the following 4 days
  </div>
  <div
    className="styled__Wrapper-ev9rqb-0 lhRYrb"
  >
    <div
      className="styled__Wrapper-e48zi3-0 bkioAu"
    >
      <div
        className="styled__Heading-e48zi3-2 eKSBRZ"
      >
        Friday
      </div>
      <div
        className="styled__DateRow-e48zi3-3 jCVlji"
      >
        4.10.2019
      </div>
      <div
        className="styled__Line-e48zi3-1 eNutno"
      >
        <span>
          12:00
        </span>
        <span
          className="styled__StyledTemp-zfc47z-0 VoffX"
        >
          18
          째C
        </span>
        <img
          className="WeatherIcon__StyledImg-qloq4m-0 kXqJBM"
          height={50}
          src="https://openweathermap.org/img/wn/null@2x.png"
          width={50}
        />
      </div>
    </div>
    <div
      className="styled__Wrapper-e48zi3-0 bkioAu"
    >
      <div
        className="styled__Heading-e48zi3-2 eKSBRZ"
      >
        Saturday
      </div>
      <div
        className="styled__DateRow-e48zi3-3 jCVlji"
      >
        5.10.2019
      </div>
      <div
        className="styled__Line-e48zi3-1 eNutno"
      >
        <span>
          12:00
        </span>
        <span
          className="styled__StyledTemp-zfc47z-0 VoffX"
        >
          18
          째C
        </span>
        <img
          className="WeatherIcon__StyledImg-qloq4m-0 kXqJBM"
          height={50}
          src="https://openweathermap.org/img/wn/null@2x.png"
          width={50}
        />
      </div>
    </div>
    <div
      className="styled__Wrapper-e48zi3-0 bkioAu"
    >
      <div
        className="styled__Heading-e48zi3-2 eKSBRZ"
      >
        Sunday
      </div>
      <div
        className="styled__DateRow-e48zi3-3 jCVlji"
      >
        6.10.2019
      </div>
      <div
        className="styled__Line-e48zi3-1 eNutno"
      >
        <span>
          12:00
        </span>
        <span
          className="styled__StyledTemp-zfc47z-0 VoffX"
        >
          18
          째C
        </span>
        <img
          className="WeatherIcon__StyledImg-qloq4m-0 kXqJBM"
          height={50}
          src="https://openweathermap.org/img/wn/null@2x.png"
          width={50}
        />
      </div>
    </div>
    <div
      className="styled__Wrapper-e48zi3-0 bkioAu"
    >
      <div
        className="styled__Heading-e48zi3-2 eKSBRZ"
      >
        Monday
      </div>
      <div
        className="styled__DateRow-e48zi3-3 jCVlji"
      >
        7.10.2019
      </div>
      <div
        className="styled__Line-e48zi3-1 eNutno"
      >
        <span>
          12:00
        </span>
        <span
          className="styled__StyledTemp-zfc47z-0 VoffX"
        >
          18
          째C
        </span>
        <img
          className="WeatherIcon__StyledImg-qloq4m-0 kXqJBM"
          height={50}
          src="https://openweathermap.org/img/wn/null@2x.png"
          width={50}
        />
      </div>
    </div>
  </div>
</div>
`;
